### DISCUSSION TITLE
Incorporating Print-Specific Features: Measurement Conversion and PDF Export Capabilities in Graphite

### DISCUSSION THREAD

**Discussion by starlord-brazil** [GraphiteEditor/Graphite#1962] [General]
Hello Graphite Developers,

I am currently working on projects intended for print and was wondering if there are any plans to include features that would allow users to switch measurement units from pixels to millimeters. Additionally, are there any developments towards integrating direct printing capabilities or generating PDF files from within Graphite? These features would greatly enhance the tool's utility for print-focused workflows. Any updates or tips on how to approach these enhancements would be highly appreciated.

Thank you!

**Comment by 0HyperCube** [comment]
Document units are covered by #586.
There is no issue for PDF export but it is on the roadmap (https://graphite.rs/features/#roadmap) under 1.0 (although if you are interested in contributing we can move this around).

I hope that helps.

**Comment by Keavon** [comment]
As Hypercube said, there are plans for PDF export and print-focused workflows but doing that correctly is difficult which is why it hasn't been prioritized as an early part of our roadmap. But if this is something you'd like to take point on and focus your development efforts around, we'd be very happy to support this. #586 would definitely be the best starting place, and I'm happy to discuss what would be involved in that if you want to talk about it on Discord, just talk about it in the #development channel and we can discuss what's involved.
